src/main/java/datawave/microservice/query/messaging/rabbitmq/config/RabbitMQMessagingConfiguration.java

```diff
@@ -66,8 +66,6 @@ public CachingConnectionFactory createCachingConnectionFactory(MessagingProperti
                 finalConnectionFactory.setPublisherConfirmType(rabbitMqProperties.getPublisherConfirmType());
             }
             
-            finalConnectionFactory.setPublisherConfirms(CachingConnectionFactory.ConfirmType.SIMPLE != rabbitMqProperties.getPublisherConfirmType()
```

Comment:
You simply removed this and did not replace it with anything.  Was this an oversight?

Reply:
`.setPublisherReConfirms()` suggests to use setPublisherConfirmType(CachingConnectionFactory.ConfirmType), which is already used above. 

As far as I can tell, `.setPublisherConfirms()` is just an extra step to call `setPublisherConfirmType()`, which is why it's deprecated I believe. Because of this, I don't think we need to replace it with anything, as it would be redundant. Thoughts?

`setPublisherConfirms()` Method Declaration:
```
	/**
	 * Use full (correlated) publisher confirms, with correlation data and a callback for
	 * each message.
	 * @param publisherConfirms true for full publisher returns,
	 * @since 1.1
	 * @deprecated in favor of {@link #setPublisherConfirmType(ConfirmType)}.
	 * @see #setSimplePublisherConfirms(boolean)
	 */
	@Deprecated
	public void setPublisherConfirms(boolean publisherConfirms) {
		Assert.isTrue(!publisherConfirms || !ConfirmType.SIMPLE.equals(this.confirmType),
				"Cannot set both publisherConfirms and simplePublisherConfirms");
		if (publisherConfirms) {
			setPublisherConfirmType(ConfirmType.CORRELATED);
		}
		else if (this.confirmType.equals(ConfirmType.CORRELATED)) {
			setPublisherConfirmType(ConfirmType.NONE);
		}
	}
```